project/MiMa.scala

```diff
@@ -456,8 +541,14 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.RemoteWatcher.receiveHeartbeatRsp"),
         ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.RemoteWatcher.selfHeartbeatRspMsg"),
 
-        // #21330 takeWhile inclusive flag
-        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.takeWhile")
+        // #21131 new implementation for Akka Typed
+        ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.actor.dungeon.DeathWatch.isWatching"),
+
+        // method create(java.lang.Class,Array[java.lang.Object])akka.actor.Props in object akka.actor.Props in current version does not have a correspondent with same parameter signature among (java.lang.Class,akka.japi.Creator)akka.actor.Props, (java.lang.Class,scala.collection.Seq)akka.actor.Props
+        ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.actor.Props.create"),
```

Comment:
I can't see how this can be right, as far as I see that `Props.create` method was introduced by Roland in 28aad82 2013-04-16 and hasn't changed signature since.